.github/workflows/build.yaml

```diff
@@ -60,7 +60,13 @@ jobs:
         uses: 'docker://zmkfirmware/zephyr-west-action-arm:latest'
         id: west-build-cyber60_rev_b
         with:
-          args: 'build "-s zmk/app -p -b cyber60_rev_b -- -DZMK_CONFIG=/github/workspace/config"'
+          args: 'build "-s zmk/app -p -b cyber60@B -- -DZMK_CONFIG=/github/workspace/config -DKEYMAP_FILE=/github/workspace/config/boards/arm/cyber60/cyber60.keymap"'
```

Comment:
We can remove this change once this PR is merged
https://github.com/zmkfirmware/zmk/pull/1015